app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def create
+    text = params[:comment]['text']
+    post_id = params[:id]
+    user_id = params[:user_id]
+    comment = Comment.new(user: current_user, post: Post.find(post_id), text:)
+    comment.increace_comments_count(post_id)
+
+    if comment.save
+      flash[:success] = 'Comment saved successfully'
+    else
+      flash.now[:error] = 'Error: Comment could not be saved'
+    end
+
+    redirect_to "/users/#{user_id}/posts/#{post_id}"
+  end
+end
```

Comment:
- [ ] [OPTIONAL] I would like to recommend you to add a **params method** because with params  inside your controller action's you can call params to access form & URL query data. Please follow [this documentation](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) to know more about :+1: